goosebit/ui/static/js/devices.js

```diff
@@ -29,6 +18,9 @@ const renderFunctions = {
         }
         return data;
     },
+    polling: (data, type) => {
+        return data ? "on time" : "overdue";
```

Comment:
Does it make sense to do the same check as before, and otherwise just return he boolean value?  Should make filtering work...
```
        if (type === "display")
```

Reply:
> Assuming we activated client-side filtering - how would it work? User entering "True" or "False"? With the current code I'd assume that if switching to client-side filtering one could use "overdue".

I meant for sorting, I don't think we want searching.

Reply:
Client-side devices sorting is not really what we would activate - but it should work with my code as well, correct? Not even sure how ascending order of a boolean is implemented.